setup.py

```diff
@@ -26,11 +26,13 @@ def read(fname):
     license="MIT",
     keywords="gpu optimizers optimization 8-bit quantization compression",
     url="https://github.com/TimDettmers/bitsandbytes",
-    install_requires=['scipy'],
     packages=find_packages(),
     package_data={"": libs},
-    install_requires=['torch', 'numpy', 'scipy'],
-    extras_require={'benchmark': ['pandas', 'matplotlib']},
+    install_requires=['torch', 'numpy'],
+    extras_require={
+        'benchmark': ['pandas', 'matplotlib'],
+        'test': ['scipy'],
```

Comment:
Is `create_normal_map` only used for testing? Otherwise the name might be confusing?

Reply:
The only call to the function is in a test (and it's mentioned in a couple copy-pasted comments too):

```
(bitsandbytes) ~/b/bitsandbytes (main) $ ag create_normal_map
tests/test_functional.py
2356:    code = F.create_normal_map()

bitsandbytes/nn/modules.py
306:        Implementation of the NF4 data type in bitsandbytes can be found in the `create_normal_map` function in

bitsandbytes/functional.py
236:def create_normal_map(offset=0.9677083, use_extra_value=True):
847:            Implementation of the NF4 data type in bitsandbytes can be found in the `create_normal_map` function ```

I can't find other relevant references to it [across all of GitHub either](https://github.com/search?q=create_normal_map+language%3APython&type=code&l=Python) – and anyway, the function is still there for downstream users who might need it, they'll just need `scipy` installed.